tests/observability/metrics/conftest.py

```diff
@@ -619,3 +620,23 @@ def aaq_resource_hard_limit_and_used(application_aware_resource_quota):
         for key, value in resource_used.items()
     }
     return formatted_hard_limit, formatted_used_value
+
+
+@pytest.fixture()
+def vm_created_pod_total_initial_metric_value(prometheus, namespace):
+    metric_query = f"sum({KUBEVIRT_VM_CREATED_BY_POD_TOTAL}{{namespace='{namespace.name}'}})"
```

Comment:
can save and reuse metric query, you can use `.format` for it, there are many examples in code, i.e.

```
tests/observability/metrics/test_vms_metrics.py:            metric_name=KUBEVIRT_VM_DISK_ALLOCATED_SIZE_BYTES.format(vm_name=windows_vm_for_test.name),
```